### DISCUSSION TITLE
Git hub Student not approved?

### DISCUSSION THREAD

**Discussion by Aarushb** [community/community#50823] [GitHub Education]
hello,
Just here to share my experience with git hub student and seek help/suggestions on how to proceed.
I tried applying for git hub student yesterday. It got denied 
Was hard enough to get a hold of a photo in the first place, since I mostly use pdf(I am visually impaired) and screen readers don't exactly play well with jpeg files, so had to figure out converting pdf and having it looked by sighted peers etc since pdf files are apparently not at all allowed to submit for some reason.
Super inconvenient, but I got it done, then I tried submitting and today got the 'request denied' email.
It said in the reason that I didn't mention my full name as it appears on my academic document on git hub profile, nor do I have a bio. Cool, I fixed both those.
Tried applying again, and now it's not letting me add the same photo....
Not only is this all I got, it was hard enough to procure in the first place but they didn't mention anything about the photo in the feedback email either and whatever was mentioned I fixed so not sure what's up here.
I wanted to reach out to git hub to explain my situation but apparently can't do that either anymore since they don't respond to student benefit related queries on their contact form.
SO here I am. Frankly very disappointed at the lack of inclusivity as well as clarity, didn't expect this from git hub at all.

**Comment by samus-aran** [comment]
Hi @Aarushb 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

We would suggest the following based on what you have described:

1. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

2. Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Please let us know how you get along.
